components/server/src/orgs/organization-service.ts

```diff
@@ -179,12 +179,14 @@ export class OrganizationService {
     }
 
     public async addOrUpdateMember(
-        userId: string,
+        userId: string | undefined, // undefined means it is a system call, not a user call
         orgId: string,
         memberId: string,
         role: OrgMemberRole,
     ): Promise<void> {
-        await this.auth.checkPermissionOnOrganization(userId, "write_members", orgId);
+        if (userId) {
```

Comment:
@svenefftinge, now thinking about that change. This could actually be the fix.

Reply:
Just curious, why this is not happening on dogfood?

Reply:
The featureflag is disabled. So these checks don't throw